packages/gatsby-cli/src/init-starter.js

```diff
@@ -67,8 +68,29 @@ const install = async rootPath => {
   process.chdir(rootPath)
 
   try {
-    let cmd = shouldUseYarn() ? spawn(`yarnpkg`) : spawn(`npm install`)
-    await cmd
+    const npmCmd = `npm install`
+    let response = npmCmd
+    if (shouldUseYarn()) {
+      const promptsAnswer = await prompts([
+        {
+          type: `select`,
+          name: `package_manager`,
+          message: `Which package manager would you like to use ?`,
+          choices: [
+            { title: `yarn`, value: `yarnpkg` },
+            { title: `npm`, value: npmCmd },
+          ],
+          max: 1,
+        },
+      ])
+      response = promptsAnswer.package_manager
```

Comment:
We need to persist the user's input using https://github.com/yeoman/configstore so that it's a one time prompt.

Reply:
I used the package.json for the [configuration](https://github.com/gatsbyjs/gatsby/pull/13225/commits/189e0ec08ce358ec204df2f56b52886c022a15a9#diff-7dc3091639b4f0b32b7271d7eb617ea8R19) was that what you expected ?